library/std/src/net/tcp.rs

```diff
@@ -53,6 +53,13 @@ use crate::time::Duration;
 ///     Ok(())
 /// } // the stream is closed here
 /// ```
+///
+/// # Platform-specific Behavior
+///
+/// On Unix, writes to the underlying socket in `SOCK_STREAM` mode are made with
+/// `MSG_NOSIGNAL` flag.
+/// This suppresses the emission of the  `SIGPIPE` signal when writing to disconnected socket.
+/// In some cases, getting a `SIGPIPE` would trigger process termination.
```

Comment:
Still needs a rewrap, unless you intended to split it into two paragraphs (which I don't think is necessary), in which case it needs an extra line.